collections/_posts/2024-05-14-organization-workspace-switcher.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Meshery support different Spaces!
+subheading: Distribute your work on different spaces 
+date: 2024-05-14
+author_contributor: Akshay Sharma
+categories: 
+  - Meshery
+  - Cloud
+permalink: /blog/organization-workspace-switcher
+featured-image: /assets/images/posts/2024-05-14-organization-workspace-switcher/organization-workspace-switcher.png
+---
+
+Meshery facilitates organization and workspace management, enabling diverse architectural spaces to thrive. Within the Layer5 Cloud ecosystem, organizations form the bedrock of multi-tenancy, while Meshery Workspaces provide virtual arenas for collaborative team endeavors. Delve deeper into the intricacies of organization and workspace dynamics <a href="https://docs.layer5.io/cloud/spaces" target="_blank">here</a>.
+
+## Spaces
+Spaces facilitate collaboration between you and your teams, allow you to control access to resources, and track activity and report on related events.Organizations can have any number of teams. Teams can have any number of users. Users can belong to any number of teams. Users may belong to any number of organizations.
+
+<a href="https://artifacthub.io" target="_blank"><img alt="Artifact Hub Kinds" src="/assets/images/posts/2024-05-14-organization-workspace-switcher/organization_units.svg" width="50%" height="80%" align="right" /></a>
+
+### Key Features
+  - **Resource Sharing**: Spaces allow for seamless resource sharing among team members, fostering collaboration.
+  - **Logical Grouping**: Within Spaces, you can group related components such as environments and infrastructure designs.
+  - **Flexibility**: Spaces support various resources like Kubernetes, Prometheus, Jaeger, Nginx, and more.
+  - **Simplified Management**: Managing and deploying resources is made easy within Spaces.
+  - **Access Control**: Spaces allow you to control access to resources by granting permissions to users and teams.
+
+We are now exposing the ability to switch between different organization and different workspace,this will enable users to :
+
+- Different spaces have different level of restriction 
+- Switch to an organization where they have the required permissions.
```

Comment:
```suggestion
- Switch to an organization where they have the required permissions.

Try Meshery's new organization and workspace switcher in the [Meshery Playground](https://play.meshery.io) and [let us know what you think](https://meshery.io/subscribe)!

```

@captain-Akshay "let us know what you think" could be a hyperlink to your blog post on the new Feedback component.